examples/bin/verify-java

```diff
@@ -42,6 +42,9 @@ Otherwise, install Java 8 or 11 in one of the following locations.
   * java (installed on PATH)
 
 Other versions of Java versions may work, but are not officially supported.
+
+For more information about selecting a Java runtime visit:
+https://druid.apache.org/docs/latest/operations/java.html#selecting-a-java-runtime
```

Comment:
How about https://druid.apache.org/docs/latest/operations/java.html? This is the first section anyway.